src/admin/settings/settings.scss

```diff
@@ -0,0 +1,25 @@
+.form-group{
+    >label{
+        white-space: normal;
+        >.checkbox{
+            position: static !important;
+        }
+    }
+    >legend{
+        @extend label;
+        text-align: right !important;
+        width: $sidebar-width !important;
+    }
+    .button-export{
+         width: $sidebar-width;
+     }
+    .button-file{
+        height: auto;
+    }
+}
+.section-title{
+    font-size: 18px;
+}
+.partial-title{
+    font-size: 16px;
+}
```

Comment:
Instead of adding .section-title and .partial-title classes, can we add these [styles to h3 & h4 elements directly? (over here)](https://github.com/motech/motech-ui/blob/master/src/common/base/base.scss#L37) and possibly use em or a variable so we don't have "magic numbers" in our CSS?